docs/config.json

```diff
@@ -850,33 +850,7 @@
         {
           "title": "Getting Started",
           "slug": "/machine-id/getting-started/"
-        },
-        {
-          "title": "Architecture",
-          "slug": "/machine-id/architecture/"
-        },
-        {
-          "title": "Troubleshooting",
-          "slug": "/machine-id/troubleshooting/"
-        },
-        {
-          "title": "FAQ",
-          "slug": "/machine-id/faq/"
-        },
-        {
-          "title": "Reference",
-          "slug": "/machine-id/reference/",
-          "entries": [
-            {
-              "title": "Configuration",
-              "slug": "/machine-id/reference/configuration/"
-            },
-            {
-              "title": "CLI",
-              "slug": "/machine-id/reference/cli/"
-            }
-          ]
-        },
+        },        
```

Comment:
Just double-checking that this just reorders sections, and doesn't change any file paths—if it does, we would need to add redirects.

Reply:
@ptgott , yes just reorg